### PR TITLE
Add a standard contributing doc

### PR DESCRIPTION
**Pull request by josephholsten**
please also review https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD to see if we're missing anything useful

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+We don't require you to use Github, and we will even take patch diffs
+attached to tickets on the issue tracker. However Github has a lot of
+convenient features, such as being able to see a diff of changes
+between a pull request and the main repository quickly without
+downloading the branch.
+
+## Functional and Unit Tests
+
+This cookbook is set up to run tests under
+[Test Kitchen](http://kitchen.ci/). It uses serverspec to run
```
Comment (shortdudey123) [inline docs/CONTRIBUTING.md:39]: some might use `inspec`

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+isn't trivial so we don't have to duplicate effort in testing. Chef
+10.14+ "doc" formatted output is sufficient.
+
+Please do indicate new platform (families) or platform versions in the
+commit message, and update the relevant ticket.  If a contribution adds
+new platforms or platform versions, indicate such in the body of the commit message(s).
+
+Please do use [foodcritic](http://www.foodcritic.io/) to
+lint-check the cookbook. Except FC007, it should pass all correctness
+rules. FC007 is okay as long as the dependent cookbooks are *required*
```
Comment (shortdudey123) [inline docs/CONTRIBUTING.md:90]: Can you elaborate on `FC007`? maybe adding a specific example would help

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (tas50) [reply docs/CONTRIBUTING.md:39]: That

**Review by tas50 — APPROVED** [review]

**Review by damacus — CHANGES_REQUESTED** [review]
serverspec --> inspec.
Elaborate on FC007

**Comment by damacus** [comment]
If this needs to be added still please add it to github.com/sous-chefs/sous-chefs/

**Pull request closed without merging**